VocaDbModel/Domain/Globalization/RegionCollection.cs

```diff
@@ -0,0 +1,260 @@
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace VocaDb.Model.Domain.Globalization {
+
+	public class RegionCollection {
+
+		public static readonly string[] RegionCodes = new string[]

```

Comment:
Wait wait wait. I don't think there's need to list them all like this. You can get the list of all cultures from .NET, [CultureInfo.GetCultures](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.getcultures?view=netframework-4.8). I'm not sure if it's "SpecificCultures" or what, but I've done it before like that.

Something like this:

```csharp
CultureInfo.GetCultures(CultureTypes.SpecificCultures).Select(culture => new RegionInfo(culture.Name).TwoLetterISORegionName).OrderBy(c => c).Distinct()
```